### DISCUSSION TITLE
为啥你的repo前面有个dotnetcore.

### DISCUSSION THREAD

**Discussion by tonyqus** [fudiwei/DotNetCore.SKIT.FlurlHttpClient.Wechat#14] [General]
想问一下你这个项目是不是加入了ncc组织？请问加入之后流量有提高吗？项目有变更有名吗？

**Comment by fudiwei** [comment]
没有组织噢。

只是一种个人的项目命名习惯：C# 项目开头加 `DotNet.` 或 `DotNetCore.`；JS/TS 项目结尾加 `.js`；Go 项目开头加 `go.`，诸如此类的。你会看到我公开的 repo 基本都是这种命名方式。

**Reply by tonyqus** [reply]
这。。。哈哈哈，好吧，可能是我敏感了。ncc的项目都有dotnetcore前缀，请忽略。
